pkg/chart/chart.go

```diff
@@ -197,12 +197,23 @@ func (t *Testing) processCharts(action func(chart string, valuesFiles []string)
 		return nil, errors.Wrap(err, "Error initializing Helm")
 	}
 
+	repoArgs := map[string][]string{}
```

Comment:
I'd say you do too much here. Please use the same logic as for `helm-extra-args` and pass the args into the `Helm` struct.

See https://github.com/helm/chart-testing/blob/master/pkg/chart/chart.go#L163

Reply:
Not sure I follow, do you agree with the way of providing different args for different repos? And the logic for parsing the args move to NewHelm with AddRepo internally select the correct args when given a repo to add?

Reply:
I guess I should have look more carefully, sorry. I'll have another look at this.